cmd/podman/containers/create.go

```diff
@@ -121,6 +121,10 @@ func commonFlags(cmd *cobra.Command) error {
 	if cmd.Flags().Changed("image-volume") {
 		cliVals.ImageVolume = cmd.Flag("image-volume").Value.String()
 	}
+
+	if cmd.Flags().Changed("gpus") {
+		cliVals.Devices = append(cliVals.Devices, "nvidia.com/gpu="+cmd.Flag("gpus").Value.String())
```

Comment:
Huh, did docker and nvidia really hardcode "GPU = nvidia"?  The [docs seem to say so](https://github.com/docker/cli/blob/acbc2540aed6f12e7f605da9be0bbbd3f6a68947/docs/reference/commandline/run.md?plain=1#L1016)...

But I can't find a reference to `nvidia.com` in the docker sources; are we sure that's not actually in the nvidia container plugin?

Looking a bit more closely, this seems to map to a `gpu` capability: https://github.com/docker/cli/blob/master/opts/gpus_test.go#L24